astropy/io/ascii/core.py

```diff
@@ -903,7 +903,7 @@ def convert_numpy(numpy_type):
     numpy_type : numpy data-type
         The numpy type required of an array returned by ``converter``. Must be a
         valid `numpy type <https://docs.scipy.org/doc/numpy/user/basics.types.html>`_,
-        e.g. numpy.int, numpy.uint, numpy.int8, numpy.int64, numpy.float,
+        e.g. int, numpy.uint, numpy.int8, numpy.int64, numpy.float,
         numpy.float64, numpy.str.
```

Comment:
What about `str`? Not mentioned in title but mentioned in the issue body.
```suggestion
        numpy.float64, str.
```

Reply:
And indeed `str` as well.